### PR TITLE
RR-436 - Wired sort controls into the view

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**

### CODE REVIEW

File: server/routes/prisonerList/prisonerListView.ts [hunk 1]
```diff
@@ -29,31 +29,42 @@ export default class PrisonerListView {
...
     }
   }
 }
 
-const buildItemsArray = (
-  pagedPrisonerSearchSummary: PagedPrisonerSearchSummary,
-  searchTerm: string,
-  statusFilter: string,
-): Item[] => {
+const buildItemsArray = (config: {
```
Comment (nathanrussell-moj-digital) [inline server/routes/prisonerList/prisonerListView.ts:46]: I've changed this to a map of config options, as the individual args were getting a bit long and messy 👍

File: server/views/pages/prisonerList/index.njk [hunk 2]
```diff
@@ -18,11 +18,8 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
       <h1 class="govuk-heading-l">Manage learning and work progress</h1>
-
-      <form class="form" method="get" novalidate="">
```
Comment (nathanrussell-moj-digital) [inline server/views/pages/prisonerList/index.njk:22]: I've moved the form back into `searchBox` - see the comments in that file to see why

File: server/views/pages/prisonerList/macros/sortableTableHeader.njk [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    #}
+    {%- set newSortOrder -%}
+      {%- if config.sortBy !== config.fieldName -%}
+        ascending
+      {%- else -%}
+        {{- 'ascending' if config.sortOrder === 'descending' else 'descending' -}}
+      {%- endif -%}
+    {%- endset -%}
+
+    <button
```
Comment (nathanrussell-moj-digital) [inline server/views/pages/prisonerList/macros/sortableTableHeader.njk:29]: @chrimesdev , I need to check with your whether the aria label and non-visible text are correct .... easier to talk you through it to explain, so lets catch up after DSU

Comment (chrimesdev) [reply server/views/pages/prisonerList/macros/sortableTableHeader.njk:29]: Agreed that we will do this as part of https://dsdmoj.atlassian.net/browse/RR-465 

Left a comment on the JIRA ticket to this comment too

**Review by chrimesdev — APPROVED** [review]

**Pull request merged**